script.js

```diff
@@ -0,0 +1,85 @@
+var userName = prompt("What is your name?");
+var foodGroup = prompt("What do you want today?\n1. Pizza\n2. Pasta\n3. Salad\nPlease write the number of your choice (1, 2, or 3).");
+var foodSubGroup;
+var foodPrice;
+var childFoodPrice;
+
+// Validate foodGroup input
+if (foodGroup === "1" || foodGroup === "2" || foodGroup === "3") {
+  switch (foodGroup) {
+    case "1":
+      foodGroup = "pizza";
+      break;
+    case "2":
+      foodGroup = "pasta";
+      break;
+    case "3":
+      foodGroup = "salad";
+      break;
+  }
+
```

Comment:
When entering another alternative than your 3 conditions, your code runs up to line 84 with the alert message "invalid food choice group" where, at line 83, you close your `switch (foodGroup)` code that starts on line 9.  Was that intended to be this apart?